src/languages/es.ts

```diff
@@ -3249,9 +3249,9 @@ const translations = {
             changeCardMonthlyLimitTypeWarning: ({limit}: CharacterLimitParams) =>
                 `Si cambias el tipo de límite de esta tarjeta a Mensual, las nuevas transacciones serán rechazadas porque ya se ha alcanzado el límite de ${limit} mensual.`,
             addShippingDetails: 'Añadir detalles de envío',
-            issuedCard: ({assignee}: AssigneeParams) => `¡emitió a ${assignee} una Tarjeta Expensify! La tarjeta llegará en 2-3 días laborables.`,
-            issuedCardNoShippingDetails: ({assignee}: AssigneeParams) => `¡emitió a ${assignee} una Tarjeta Expensify! La tarjeta se enviará una vez que se agreguen los detalles de envío.`,
-            issuedCardVirtual: ({assignee, link}: IssueVirtualCardParams) => `¡emitió a ${assignee} una ${link} virtual! La tarjeta puede utilizarse inmediatamente.`,
+            issuedCard: ({assignee}: AssigneeParams) => `¡Emitió a ${assignee} una Tarjeta Expensify! La tarjeta llegará en 2-3 días laborables.`,
```

Comment:
NAB for this issue but based on the comment [here](https://github.com/Expensify/App/pull/46281#discussion_r1694932272) it looks like the starting character `i` in Spanish translation has come in uninvited. If so, let us also fix it here. 
@Gonals What do you think?

Reply:
> I think it is intentional. The symbol "¡" is an inverted exclamation mark, used in Spanish to indicate the beginning of an exclamatory sentence or phrase

Oh ok. That's interesting. Let's proceed then if it's intentional.

Reply:
TBH I don't think it's a bug as I see some other cases that we do not start with an uppercased letter, yet we omit the action's actor name
<img width="450" alt="Screenshot 2024-11-04 at 20 26 57" src="https://github.com/user-attachments/assets/74b6f557-a51b-4b87-9a55-a991e247f8c8">

so the message is kind of a logical continuation: [User] [did something]

for the exclamation mark - not a Spanish expert, but AFAIK it can be either at the beginning of the sentence or just a clause, but now I'm not sure if the scheme [User] ¡[did something]! is correct. Looks like it should be before the user name, but it's not possible in this case.

I think it should be consulted with the design team and Spanish translation should be approved

Reply:
> I think it should be consulted with the design team and Spanish translation should be approved

Thanks @koko57 for your feedback. I have raised the issue in slack [here](https://expensify.slack.com/archives/C01GTK53T8Q/p1730728522184819). Let us see what the Spanish translation team think about this.

Reply:
answering to that thread: I know it's intentional, but what I was pointing out here is that we shouldn't capitalize this first letter. Like in the example I've given above. And if we won't capitalize it this inverted exclamation mark is probably incorrect in this case

UserName
¡emitió a ${assignee} una Tarjeta Expensify!

Reply:
> TBH I don't think it's a bug as I see some other cases that we do not start with an uppercased letter, yet we omit the action's actor name <img alt="Screenshot 2024-11-04 at 20 26 57" width="450" src="https://private-user-images.githubusercontent.com/28020445/382771236-74b6f557-a51b-4b87-9a55-a991e247f8c8.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzA3MzA5ODYsIm5iZiI6MTczMDczMDY4NiwicGF0aCI6Ii8yODAyMDQ0NS8zODI3NzEyMzYtNzRiNmY1NTctYTUxYi00Yjg3LTlhNTUtYTk5MWUyNDdmOGM4LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDExMDQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMTA0VDE0MzEyNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWE1N2Q3MGZiNTg1ZDdhNDgyOTkyMTYzYWU4ZjNiYTQ3NzliYmFkOTMwZmY1ZTlhZGM1Y2E4ZGY1MmM1NWYxODMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.IlVWf6UixzeI3Ty09l_GlyUp9FWuBsO2Uy9nJRt_TU4">
> 
> so the message is kind of a logical continuation: [User] [did something]

Hmm… Sorry. I missed your point but I get it now. Thanks. Let me get the design team involved here.

@Expensify/Design In this issue, we are capitalising `issued` to `Issued` as shown below:
But now I am also wondering if this needs to be done due to the reasoning above.
What do you think?

![Screenshot 2024-11-04 at 8 17 25 PM](https://github.com/user-attachments/assets/2c0d3709-8ac0-409a-b1ca-ee85979043d1)

Reply:
Yeah I'm pretty sure for system messages like this, we do not capitalize the first letter. Right @trjExpensify ?

Reply:
Yeah, ordinarily for a one sentence sys message. Though I think these ones where there's two sentences, we might capitalise. I seem to recall @jamesdeanexpensify setting a rule for that or something.

Reply:
Typically it has been no capital at the beginning of any system messages. But if the system message is more than one sentence, then you would capitalize the beginning of the second sentence. Does that make sense?

Reply:
>Typically it has been no capital at the beginning of any system messages.

Right, because system messages are written with the user's name as the subject of the sentence.

> **James Dean**
> did something to the expense.

>**James Dean**
> did something to the expense. Previously it was different.

Reply:
Ah cool, so then I'm misremembering that convo and Shawn's suggestion is correct. 👍

Reply:
> for system messages like this, we do not capitalize the first letter

Ok. So that means that the changes in this PR i.e. `i` to `I` is not required. cc @robertjchen

Reply:
Got it, let's close this out then 👍